src/util/directiontables.h

```diff
@@ -31,7 +31,14 @@ extern const v3s16 g_26dirs[26];
 // 26th is (0,0,0)
 extern const v3s16 g_27dirs[27];
 
-extern const u8 wallmounted_to_facedir[6];
+constexpr const u8 wallmounted_to_facedir[6] = {
```

Comment:
It's just a header, so `extern constexpr const u8 wallmounted_to_facedir[6];` isn't valid?

Reply:
if its a constexpr it have to initialized immediately, so its invalid.